### PR TITLE
users/me is deprecated and replaced by users/self

### PR DESCRIPTION
**Pull request by Astro-Choco**
The updates for the deprecated users/me are as follows:

- Updated all occurrences of users/me and replaced them with users/self
- Added a change entry in CHANGELOG

Related JIRA ticket: DEVX-713.

### CODE REVIEW

**Review by davidruiz14 — COMMENTED** [review]
All looks good. Great change of wording on the explanation of the path parameter.

**Review by sherifzain — APPROVED** [review]
LGTM!

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -7,6 +7,20 @@ Each log entry uses the following tags:
 - For API changes and the corresponding documentation changes, the log is tagged with the name of the changes API. For example, `OAuth API`.
 - For documentation changes that are editorial and are not introduced by any API change, the log is tagged with `Editorial changes`.
 
+.**January 18, 2022** `OAuth API` `Editorial changes`
+[%collapsible]
+====
+**Replaced the deprecated endpoint `users/me` with `users/self`**
+
+The following updates are made for the deprecation:
```
Comment (AndreaFilyo) [inline CHANGELOG.adoc:15]:
```suggestion
The following updates were done for the deprecation:
```

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -7,6 +7,20 @@ Each log entry uses the following tags:
...
+
+- The OAuth API endpoint `users/me` is deprecated and replaced by `users/self`.
+- Until `users/me` is removed, all API requests to it will be redirected to `users/self`.
+- The API documentation is updated to include `users/self` instead of `users/me`.
+
+> **Note:** The support for the deprecated `users/me` will continue until further notice. Please replace `users/me` to `users/self` in your integration as soon as possible.
```
Comment (AndreaFilyo) [inline CHANGELOG.adoc:21]:
```suggestion
> **Note:** The support for the deprecated `users/me` will continue until further notice. It is recommended to replace `users/me` with `users/self` in your integration as soon as possible.
```

**Review by AndreaFilyo — APPROVED** [review]
Looks good, only added a couple of minor suggestions.

File: faq.adoc [hunk 3]
```diff
@@ -81,10 +81,10 @@ filter instead of `{organizationUuid}`
 [%collapsible]
 ====
 ****
-You can either use the `self` filter or call the following endpoint to fetch the organization UUID.
+You can use the `self` path parameter instead of specifying organization UUID when calling the APIs. Or, you can call the following endpoint to fetch the organization UUID.
```
Comment (anand-izettle) [inline faq.adoc:84]: can you please change "path parameter" to "endpoint". Path parameter is name for a variable in an endpoint url not the url itself.

**Review by anand-izettle — CHANGES_REQUESTED** [review]
can you please change the faq.adoc.. everything else looks great.. :)

File: faq.adoc (hunk 3 above)
Comment (Astro-Choco) [reply faq.adoc]: Thanks! Fixed it.

**Review by anand-izettle — APPROVED** [review]

**Pull request merged**